src/components/Alert/Alert.d.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+declare type AlertType =
+| "primary"
+| "secondary"
+| "success"
+| "info"
+| "warning"
+| "danger";
+
+declare namespace Alert {
+    export interface AlertProps extends React.Props<Alert> {
```

Comment:
It's really not that big an issue but the `React.Props` type is being deprecated. Prefer `React.HTMLProps`.